common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1005,6 +1005,12 @@
     ],
     "sqlState" : "42702"
   },
+  "EXEC_IMMEDIATE_DUPLICATE_ARGUMENT_ALIASES" : {
+    "message" : [
+      "Using statement contains multiple arguments with same alias (<aliases>)."
+    ],
+    "sqlState" : "42702"
```

Comment:
Thinking about this... why is this a problem? It's a problem in INTO (for which we would use 42701